### PR TITLE
Updating sample C# snippets and code

### PR DESCRIPTION
**Pull request by rangothedog**
Updating sample C# snippets and code.

### CODE REVIEW

File: sdk/translation/Azure.AI.Translation.Text/samples/Sample0_CreateClient.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Once you have the value for the API key, create an `AzureKeyCredential`.
 
 With the value of the `AzureKeyCredential`, you can create the [TextTranslationClient][translator_client_class]:
 
-```C#
-AzureKeyCredential credential = new("<apiKey>");
-TextTranslationClient client = new(credential);
+```C# Snippet:CreateTextTranslationClientWithKey
+string apiKey = "<Text Translator Resource API Key>";
+var client = new TextTranslationClient(new AzureKeyCredential(apiKey));
```
Comment (MikeyMCZ) [inline sdk/translation/Azure.AI.Translation.Text/samples/Sample0_CreateClient.md:17]: You are using `var` here for the Type of the variable. When I was working on samples initially, I have noticed that usually type is defined - probably for better readability, can you please double check that and keep what other projects are using?

File: sdk/translation/Azure.AI.Translation.Text/tests/Samples/SampleSnippets.cs [hunk 2]
```diff
@@ -38,10 +40,102 @@ public TextTranslationClient CreateTextTranslationClient()
...
+        [Test]
+        public TextTranslationClient CreateTextTranslationClientWithKey()
+        {
+            #region Snippet:CreateTextTranslationClientWithKey
+
+#if SNIPPET
+            string apiKey = "<Text Translator Resource API Key>";
+#else
+            string apiKey = TestEnvironment.ApiKey;
+
```
Comment (MikeyMCZ) [inline sdk/translation/Azure.AI.Translation.Text/tests/Samples/SampleSnippets.cs:52]: nit: This space that you have included before endif is not consistent. See line 124. Can you please remove it from all occurrences?

File: sdk/translation/Azure.AI.Translation.Text/samples/Sample0_CreateClient.md (hunk 1 above)
Comment (rangothedog) [reply sdk/translation/Azure.AI.Translation.Text/samples/Sample0_CreateClient.md:17]: Agreed, will change it.

File: sdk/translation/Azure.AI.Translation.Text/tests/Samples/SampleSnippets.cs (hunk 2 above)
Comment (rangothedog) [reply sdk/translation/Azure.AI.Translation.Text/tests/Samples/SampleSnippets.cs:51]: Removed.

**Pull request merged**